docs/configuration/index.md

```diff
@@ -1769,14 +1769,11 @@ If there is an L1 miss and L2 hit, it will also populate L1.
 
 This section describes configurations that control behavior of Druid's query types, applicable to Broker, Historical, and MiddleManager processes.
 
-### Query vectorization config
+### Overriding default query context values
 
-The following configurations are to set the default behavior for query vectorization.
-
-|Property|Description|Default|
-|--------|-----------|-------|
-|`druid.query.vectorize`|See [Vectorization parameters](../querying/query-context.html#vectorization-parameters) for details. This value can be overridden by `vectorize` in the query contexts.|`true`|
-|`druid.query.vectorSize`|See [Vectorization parameters](../querying/query-context.html#vectorization-parameters) for details. This value can be overridden by `vectorSize` in the query contexts.|`512`|
+Any [Query Context General Parameter](../querying/query-context.html#general-parameters) default value 
+can be overridden by setting runtime property in the format of `druid.query.override.default.context.{query_context_key}`. 
+Note that the runtime property value can be overridden if value for the same key is explicitly specify in the query contexts.
```

Comment:
Suggest adding a description of the precedence chain here, like what you have in the PR description:

> The order of precedence is as follow: hard_coded_default_value <- runtime_properties <- query_context_parameter

Reply:
Done